app/uk/gov/hmrc/transitmovementsrouter/models/ObjectStoreRetentionPeriod.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.transitmovementsrouter.models
+
+import uk.gov.hmrc.objectstore.client.RetentionPeriod
+
+case class ObjectStoreRetentionPeriod(value: RetentionPeriod) extends AnyVal
```

Comment:
Why are you wrapping this in an `AnyVal`? `RetentionPeriod` is already a specific type so there is no need to wrap it. We wrap `String`s in `AnyVal`s because they are easy to put in the wrong place.

Remove this `AnyVal` and just keep the `RetentionPeriod`.